plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/render/displayAdapter/impl/EventHandler.java

```diff
@@ -60,19 +66,22 @@ public void handleEvent( Event event ) {
             break;
         case SWT.Resize:
             controlResized(event);
+            break;
```

Comment:
I assume we have no BUG here since other cases haven't implemented anything. So nice catch to avoid BUGS in the future!